pkgs/tools/networking/passh/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "passh";
+  version = "2020-03-10";
```

Comment:
I asked on IRC, and it appears best practice regarding unversioned packages is to set it to be the date of the last commit, not the date the package is created.

Reply:
Fixed :)